draft-irtf-cfrg-hpke.md

```diff
@@ -754,38 +799,38 @@ def OpenAuthPSK(enc, skR, info, aad, ct, psk, pskID, pkS):
 | 0x0010 | DHKEM(P-256, HKDF-SHA256)      | 32   | 65   | 65  | {{NISTCurves}}, {{?RFC5869}} |
 | 0x0011 | DHKEM(P-384, HKDF-SHA384)      | 48   | 97   | 97  | {{NISTCurves}}, {{?RFC5869}} |
 | 0x0012 | DHKEM(P-521, HKDF-SHA512)      | 64   | 133  | 133 | {{NISTCurves}}, {{?RFC5869}} |
-| 0x0020 | DHKEM(Curve25519, HKDF-SHA256) | 32   | 32   | 32  | {{?RFC7748}}, {{?RFC5869}}   |
-| 0x0021 | DHKEM(Curve448, HKDF-SHA512)   | 64   | 56   | 56  | {{?RFC7748}}, {{?RFC5869}}   |
+| 0x0020 | DHKEM(X25519, HKDF-SHA256) | 32   | 32   | 32  | {{?RFC7748}}, {{?RFC5869}}   |
+| 0x0021 | DHKEM(X448, HKDF-SHA512)   | 64   | 56   | 56  | {{?RFC7748}}, {{?RFC5869}}   |
 
 ### Marshal
 
-For the NIST curves P-256, P-384 and P-521, the Marshal function of the
-DH scheme produces the normal (non-compressed) representation of the
-public key, according to {{SECG}}.
+For the groups based on NIST curves P-256, P-384 and P-521, the Marshal function
```

Comment:
This doesn't make sense.  The curve is a group.

Reply:
This is just for consistency with the X25519/curve25519 notation. I think this is more clear, and lets us speak in terms of DH groups rather than specific curves.

Reply:
I don't think we need to distinguish between the curve group and the subgroup here.  It's not ambiguous.

Reply:
Resolved with less text!